compositions/aws-provider/eks/eks-managed-node-group-subnet-labels.yaml

```diff
@@ -75,13 +75,11 @@ spec:
             - type: string
               string:
                 fmt: "%s-eks-cluster-conn"
+        - type: ToCompositeFieldPath
+          fromFieldPath: metadata.name
+          toFieldPath: status.clusterName
         - fromFieldPath: spec.writeConnectionSecretToRef.namespace
           toFieldPath: spec.writeConnectionSecretToRef.namespace
-#        - type: ToCompositeFieldPath
-#          fromFieldPath: status.atProvider.identity.oidc.issuer
-#          toFieldPath: status.eks.oidc
-#          policy:
-#            fromFieldPath: Optional
```

Comment:
Do you know why these fields we commented out? And why we are removing them vs uncommenting?

Reply:
Not too sure tbh. I think this was left over from our initial commit. As far as I am aware, it hasn't been used.